package.json

```diff
@@ -40,6 +40,7 @@
         "react-redux": "^7.2.2",
         "react-router-dom": "^5.2.0",
         "react-scripts": "4.0.3",
+        "react-uuid": "^1.0.2",
```

Comment:
See this - https://blog.bitsrc.io/why-is-nanoid-replacing-uuid-1b5100e62ed2

Reply:
Agree - changed